goodskin/theme.json

```diff
@@ -0,0 +1,874 @@
+{
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"background": true,
+			"custom": true,
+			"customDuotone": true,
+			"customGradient": true,
+			"defaultDuotone": false,
+			"defaultGradients": false,
+			"defaultPalette": false,
+			"palette": [
+				{
+					"color": "#ffffff",
+					"name": "Base",
+					"slug": "base"
+				},
+				{
+					"color": "#000000",
+					"name": "Contrast",
+					"slug": "contrast"
+				},
+				{
+					"color": "#000000",
+					"name": "Primary",
+					"slug": "primary"
+				},
+				{
+					"color": "#efefea",
+					"name": "Background",
+					"slug": "custom-background"
+				},
+				{
+					"color": "#f9d7ff",
+					"name": "Accent",
+					"slug": "custom-accent"
+				}
+			],
+			"text": true
+		},
+		"layout": {
+			"contentSize": "780px",
+			"wideSize": "1020px"
+		},
+		"shadow": {
+			"defaultPresets": true,
+			"presets": []
+		},
+		"spacing": {
+			"customSpacingSize": true,
+			"defaultSpacingSizes": false,
+			"spacingSizes": [
+				{
+					"name": "1",
+					"size": "0.5rem",
+					"slug": "30"
+				},
+				{
+					"name": "2",
+					"size": "1.25rem",
+					"slug": "40"
+				},
+				{
+					"name": "3",
+					"size": "clamp(1.75rem, 2.2896394vw, 2.5rem)",
+					"slug": "50"
+				},
+				{
+					"name": "4",
+					"size": "clamp(2.5rem, 4.5792788vw, 5rem)",
+					"slug": "60"
+				},
+				{
+					"name": "5",
+					"size": "clamp(3.75rem, 6.8689181vw, 7.5rem)",
+					"slug": "70"
+				},
+				{
+					"name": "6",
+					"size": "clamp(5rem, 9.1585575vw, 10.0rem)",
+					"slug": "80"
+				}
+			],
+			"units": [
+				"%",
+				"px",
+				"em",
+				"rem",
+				"vh",
+				"vw"
+			]
+		},
+		"typography": {
+			"customFontSize": true,
+			"defaultFontSizes": false,
+			"dropCap": true,
+			"fluid": true,
+			"fontFamilies": [
+				{
+					"fontFace": [
+						{
+							"fontFamily": "\"Instrument Serif\"",
+							"fontStyle": "normal",
+							"fontWeight": "400",
+							"src": "https://skinblog.mystagingwebsite.com/wp-content/fonts/jizBRFtNs2ka5fXjeivQ4LroWlx-2zcZj1bIkNo.woff2"
```

Comment:
Use local asset reference. 
You may need to save theme changes before export if you are using CBT